### PR TITLE
5768 MOVE(continue) assumed and request fixes

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#5768

*Description of changes:*

* This is a fix for MVE requests. Currently they were not working as XPRO, so we have now mapped (CA AND IN) back to BC.. so we can see the BC entities and make a BC request. 

* This also fixes the assumed named flow for MVE requests, it will now follow the normal BC flow.

*Notes:*
* Personally if we map back to BC we should have made the location selection BC and updated the verbiage, but this is what was asked for..

* This can create an issue because we have an INTERNATIONAL Selection that essentially does nothing.. and a Canada selection that maps back to BC.. 

* This is a quick(NOT QUICK) fix that will allow the request to complete but designers and BA's need to look at this issue and come up with a long term solution as i don't like this much..

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Review by vysakh-menon-aot — APPROVED** [review]

File: client/src/components/new-request/search.vue [hunk 1]
```diff
@@ -343,6 +343,7 @@ export default class NewSearch extends Vue {
     newReqModule.mutateIsPersonsName(value)
   }
   get location () {
+    if (['MVE'].includes(this.request_action_cd)) newReqModule.mutateLocation('BC')
```
Comment (severinbeauvais) [inline client/src/components/new-request/search.vue:346]: Could you please add code comments explaining things here?

File: client/src/store/new-request-module.ts [hunk 2]
```diff
@@ -245,6 +245,13 @@ export class NewRequestModule extends VuexModule {
         `Reported separately as Corporate tax`,
         `Has name protection in BC`
       ],
+      mveBlurbs: [
+        `A company that may have one or more people who own shares with some personal responsibility for
+        debt and liabilities.`,
```
Comment (severinbeauvais) [inline client/src/store/new-request-module.ts:250]: I think "debt" should be plural (as on line 281).

File: client/src/store/new-request-module.ts [hunk 3]
```diff
@@ -376,6 +403,12 @@ export class NewRequestModule extends VuexModule {
         'Reported as Corporate tax',
         'Has name protection in BC'
       ],
+      mveBlurbs: [
+        'A type of corporations which has a benefit to  the community.  They are intended to bridge the gap between ' +
```
Comment (severinbeauvais) [inline client/src/store/new-request-module.ts:407]: "corporations" should be singular?

Also, should be "xxx, which yyy" or "xxx that yyy", if possible. (Here, I prefer the "that" form.)

Second sentence refers to "they", which is plural. Can you use "it" to solve this?

File: client/src/store/new-request-module.ts [hunk 4]
```diff
@@ -399,6 +432,14 @@ export class NewRequestModule extends VuexModule {
         'Has name protection in BC',
         'Must use Societies Online to register a name and incorporate'
       ],
+      mveBlurbs: [
+        `A non-profit organization that is also known as a non-share corporation.`,
+        'Any funds or profits must be used only for social or community benefit',
+        'When incorporated has independent legal status separate from its members',
```
Comment (severinbeauvais) [inline client/src/store/new-request-module.ts:438]: Comma after "incorporated".

**Review by severinbeauvais — APPROVED** [review]
Looks ok. Just a few comments on existing grammar :)

File: client/src/components/new-request/search.vue (hunk 1 above)
Comment (cameron-freshworks) [reply client/src/components/new-request/search.vue:351]: Definitely!

File: client/src/store/new-request-module.ts (hunk 2 above)
Comment (cameron-freshworks) [reply client/src/store/new-request-module.ts:250]: for sure!

File: client/src/store/new-request-module.ts (hunk 3 above)
Comment (cameron-freshworks) [reply client/src/store/new-request-module.ts:407]: lol for sure :)

File: client/src/store/new-request-module.ts (hunk 4 above)
Comment (cameron-freshworks) [reply client/src/store/new-request-module.ts:438]: Will add!

**Pull request closed without merging**